### ISSUE TITLE
Use of torch2trt prompts `known unsupported method` warning

### ISSUE THREAD

**Issue by Hommus** [NVIDIA-AI-IOT/torch2trt#685]
In an attempt to optimise inference on a Jetson Xavier, I'm using torch2trt as follows:
```python
from torch2trt import torch2trt
...
self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
self.model.eval().to(self.device)
x = torch.ones((1, 3, 200, 320), device=self.device)
self.model = torch2trt(self.model, [x])
...
prediction = self.model(inputs)
```
However, I'm receiving the following error, and also a significant decrease in model accuracy:
`Warning: Encountered known unsupported method torch.nn.functional.has_torch_function_unary
`

What are the steps I should follow to debug this error or potential problems?

PyTorch version: 1.10.0
TensorRT version: 7.1.3
Platform (ie: Jetson Nano): Jetson Xavier
